app/frontend/courses/report/types/CoursesReport__StudentOverview.res

```diff
@@ -75,6 +79,7 @@ let averageQuizScore = t =>
 
 let makeFromJs = (id, studentData) => {
   id: id,
+  cohort: studentData["student"]["cohort"]->Cohort.makeFromFragment,
```

Comment:
How does this work? 🤔 

You are only fetching the name of the cohort; 
```
        student {
          cohort {
            name
          }
        }
```

The issue here is that we are bypassing the type system check; Please convert this query into a new format so that we don't make this error going forward; 

Example

```res
module CohortDetailsDataQuery = %graphql(`
  query CohortDetailsDataQuery($id: ID!) {
    cohort(id: $id) {
      ...CohortFragment
    }
  }
`)


  CohortDetailsDataQuery.fetch(
    ~notifyOnNotFound=false,
    {
      id: id,
    },
  )
  |> Js.Promise.then_((response: CohortDetailsDataQuery.t) => {
    setState(_ => Loaded(response.cohort->Cohort.makeFromFragment))
    setCourseId(response.cohort.courseId)
    Js.Promise.resolve()
  })
  |> Js.Promise.catch(_error => {
    setState(_ => Errored)
    Js.Promise.resolve()
  })
  |> ignore
```